cabal-install/src/Distribution/Client/Setup.hs

```diff
@@ -1106,6 +1107,8 @@ filterTestFlags flags cabalLibVersion
       flags_latest
         { -- Cabal < 3.0 doesn't know about --test-wrapper
           Cabal.testWrapper = NoFlag
+        , Cabal.testCoverageLibsModules = NoFlag
+        , Cabal.testCoverageDistPrefs = NoFlag
```

Comment:
The flag has to be removed from `flags_3_11_0`. See the comments above for an explanation, and see `filterConfigureFlags` for how to add a new `flags_` (they build on each other).

Reply:
Done.